set.mm

```diff
@@ -555055,7 +555051,8 @@ relation with an intersection with the same Cartesian product (cf.
       APZUFTABCUEDRUFUHUIACDSUAUB $.
   $}
 
-  $( Lemma for ~~? trcoss .  (Contributed by Peter Mazsa, 2-Oct-2018.) $)
+  $( Sufficient condition for transitivity of conjunctions inside existential
+     quantifiers.  (Contributed by Peter Mazsa, 2-Oct-2018.) $)
   motr $p |- ( E* x ps ->
                ( ( E. x ( ph /\ ps ) /\ E. x ( ps /\ ch ) ) ->
```

Comment:
rename "moantr".

Reply:
Is there a reason why the theorem should not be renamed as proposed? I think an indication that it is a theorem about conjunctions ("and") should be given.

Reply:
@avekens No I thought I did it, but not: now I renamed it, thank you.